src/vs/workbench/contrib/debug/browser/debugEditorActions.ts

```diff
@@ -133,6 +134,33 @@ class LogPointAction extends EditorAction2 {
 	}
 }
 
+class GoToDisassemblyAction extends EditorAction {
+
+	public static readonly ID = 'editor.debug.action.goToDisassembly';
+	public static readonly LABEL = nls.localize('goToDisassembly', "Go to Disassembly");
+
+	constructor() {
+		super({
+			id: GoToDisassemblyAction.ID,
+			label: GoToDisassemblyAction.LABEL,
+			alias: 'Debug: Go to Disassembly',
+			precondition: ContextKeyExpr.and(CONTEXT_IN_DEBUG_MODE, PanelFocusContext.toNegated(), CONTEXT_DEBUG_STATE.isEqualTo('stopped'), EditorContextKeys.editorTextFocus),
```

Comment:
I suggest to use the context here if the dissasembly is supported by the focused session.

Reply:
Sorry but I'm not sure I know what you mean?